src/ResumeCache.cpp

```diff
@@ -0,0 +1,12 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#include "src/ResumeCache.h"
+
+namespace reactivesocket {
+
+bool ResumeCache::retransmitFromPosition(
+    position_t initialPosition, ConnectionAutomaton &connection, const RetransmitFilter &filter) {
```

Comment:
The RetransmitFilter is the best way I can envision now for allowing individual streams to be ERRORed or CANCELed during resume.